### PR TITLE
Update Documentation in README.md

### PR DESCRIPTION
**Pull request by JCBird1012**
This PR has numerous clarifications for the `Setting Up` in `README.md`, as well as other typo fixes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,25 +6,22 @@ Check it out in action at [shuttles.rpi.edu](https://shuttles.rpi.edu).
 
 ## Setting Up
 
-1. Install Go
-2. `go get github.com/wtg/shuttletracker`
-3. `govendor sync`
-4. Make sure you have npm, bower, golang and mongodb installed
-5. Run `bower install` inside shuttle tracking directory to install dependencies listed in bower.json
-6. Rename conf.json.sample to conf.json
+1. Install Golang (https://golang.org/doc/install)
```
Comment (kochman) [inline README.md]: Change "Golang" to "Go"?

File: README.md [hunk 2]
```diff
@@ -6,25 +6,22 @@ Check it out in action at [shuttles.rpi.edu](https://shuttles.rpi.edu).
...
+2. Run `go get github.com/wtg/shuttletracker`
+3. Run `govendor sync`
```
Comment (kochman) [inline README.md]: Should we include instructions for installing `govendor`?

i.e. `go get -u github.com/kardianos/govendor`

File: README.md [hunk 3]
```diff
@@ -6,25 +6,22 @@ Check it out in action at [shuttles.rpi.edu](https://shuttles.rpi.edu).
...
-   * MongoPort: Port where MongoDB is bound (default is 27017)
-9. Run `bower install` inside shuttle tracking directory to install dependencies listed in bower.json
-10. Rename conf.json.sample to conf.json and edit with the following:
-   * Data Feed: API with tracking information (iTrak in our case), if using the dummy server, http://localhost:8081
-   * UpdateInterval: Number of seconds between each request to the data feed
-   * MongoUrl: Url where MongoDB is located
-   * MongoPort: Port where MongoDB is bound (default is 27017)
-11. Run the app using `go run main.go` in the project root directory
-12. Visit http://localhost:8080/ to view the tracking application and http://localhost:8080/admin to view the admin panel
+   * `Data Feed`: API with tracking information from iTrak... For RPI, this is a unique API URL that we can get data from. It's currently private, and we will only share it with authorized members for now.
```
Comment (kochman) [inline README.md]: "Data Feed" -> "DataFeed"

File: README.md (hunk 2 above)
Comment (JCBird1012) [reply README.md]: Definitely. Totally overlooked that.

**Comment by JCBird1012** [comment]
I've made the suggested changes, and added a bit about $GOPATH

**Comment by kochman** [comment]
Thanks Jonathan!

**Pull request merged**